File: myClass.m

#import <Foundation/Foundation.h>

@interface myClass : NSObject 


@end

@implementation myClass

-(void)Print
{
    NSLog(@"Halló Daniel!");
}

@end


int main(int argc, const char * argv[]) {
    @autoreleasepool {
        myClass *c = [myClass init];
        [c Print];
    }
    return 0;
}

File: myClass.swift
import Foundation

class myClass{
    func Main()
    {
        var str = "Halló ";
        str.AddDaniel()
        print(str);
    }
}

extension String{
    mutating func AddDaniel()
    {
        self = self + "Daniel";
    }
}

let c = myClass();

c.Main();